index.html

```diff
@@ -232,6 +232,53 @@ <h2>About me</h2>
       </div>
     </div>
   </section>
+  <footer>
+    <div class="footer-section">
+    <div>
+      <p class="footer-p">
+        I'm always interested in hearing about new projects,
+         so if you'd like to chat please get in touch.
+      </p>
+    </div>
+   <div>
+    <form action="https://formspree.io/f/mwkzggnv" method="post">
+      <input type="text" placeholder="Full name" maxlength="30" id="text" name="text" required>
+      <input type="email" id="email" name="email" placeholder="Email address" maxlength="30" required>
+      <textarea name="" id="" cols="10" rows="5" required maxlength="500" name="textarea" id="textarea"  placeholder="Enter text here"></textarea>
+      <button class="Multi-Post-project button form-button" type="submit">Get in touch</button>
+    </form>
```

Comment:
- Good job using `flex` for the form as the previous reviewer suggested. However, doing so resulted in changing the design. Kindly do the following:
   - [ ] Increasing the spacing around the form section to match the Figma template.
   - [ ] Centering the "Get in touch" button to match the given design.
   

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/220341587-86ed97df-d8f4-4848-a64a-b0f201d6ea31.png) | ![image](https://user-images.githubusercontent.com/62290545/220341502-54798716-7781-41f4-9451-4256d40ffcbe.png) |